golangci-lint: error handling

```bash
2024/06/03 02:18:03.135392 [67c45db0-214f-11ef-8d6c-cc66f8605ff1][ERROR] linters.go:127: golangci-lint run failed: exit status 3, mark and continue
2024/06/03 02:18:03.135514 [67c45db0-214f-11ef-8d6c-cc66f8605ff1][WARN] linters.go:250: unexpected linter check output: Error: parallel golangci-lint is running
2024/06/03 02:18:03.135561 [67c45db0-214f-11ef-8d6c-cc66f8605ff1][WARN] linters.go:250: unexpected linter check output: Failed executing command with error: parallel golangci-lint is running
2024/06/03 02:18:03.135577 [67c45db0-214f-11ef-8d6c-cc66f8605ff1][INFO] linters.go:172: [golangci-lint] found total 0 files with 0 lint errors on repo qbox/mars
2024/06/03 02:18:03.135607 [67c45db0-214f-11ef-8d6c-cc66f8605ff1][INFO] linters.go:179: [golangci-lint] found 0 files with valid 0 linter errors related to this PR 2624 (qbox/mars)
2024/06/03 02:18:04.223831 [67c45db0-214f-11ef-8d6c-cc66f8605ff1][INFO] linters.go:195: [golangci-lint] create check run success, HTML_URL: https://github.com/qbox/mars/runs/25712593609
2024/06/03 02:18:07.468547 [753b7aa0-214f-11ef-9020-d61989f9f07f][ERROR] linters.go:127: golangci-lint run failed: exit status 3, mark and continue
2024/06/03 02:18:07.468680 [753b7aa0-214f-11ef-9020-d61989f9f07f][WARN] linters.go:250: unexpected linter check output: Error: parallel golangci-lint is running
2024/06/03 02:18:07.468737 [753b7aa0-214f-11ef-9020-d61989f9f07f][WARN] linters.go:250: unexpected linter check output: Failed executing command with error: parallel golangci-lint is running
```

--- a/config/config.yaml
+++ b/config/config.yaml
@@ -11,7 +11,7 @@ customConfig: # custom config for specific orgs or repos
   qbox: # github organization name
     golangci-lint:
       enable: true
-      args: ["run", "-D", "staticcheck"] # disable staticcheck globally since we have a separate linter for it
+      args: ["run", "-D", "staticcheck", "--allow-parallel-runners=true","--timeout=5m0s"] 
 
   qbox/net-cache:
     luacheck:
@@ -20,11 +20,11 @@ customConfig: # custom config for specific orgs or repos
   qbox/net-cache:
     golangci-lint:
       enable: true
-      args: ["run", "--enable-all","-D", "staticcheck", "--timeout","5m0s"] # disable staticcheck globally since we have a separate linter for it      
+      args: ["run", "--enable-all","-D", "staticcheck", "--timeout=5m0s", "--allow-parallel-runners=true"] # disable staticcheck globally since we have a separate linter for it      
   qbox/net-gslb:
     golangci-lint:
       enable: true
-      args: ["run", "--enable-all","-D", "staticcheck","--timeout","5m0s"] # disable staticcheck globally since we have a separate linter for it      
+      args: ["run", "--enable-all","-D", "staticcheck","--timeout=5m0s", "--allow-parallel-runners=true"] # disable staticcheck globally since we have a separate linter for it      
 
   qbox/kodo:
     staticcheck:

--- a/docs/website/docs/components/go/golangci-lint.md
+++ b/docs/website/docs/components/go/golangci-lint.md
@@ -3,17 +3,17 @@ title: golangci-lint
 sidebar_position: 1
 ---
 
-[golangci-lint](https://github.com/golangci/golangci-lint) 是go语言领域非常优秀的linters执行器，她内置支持了很多go领域linter工具。
+[golangci-lint](https://github.com/golangci/golangci-lint) 是 go 语言领域非常优秀的 linters 执行器，她内置支持了很多 go 领域 linter 工具。
 
-**Reviewbot** 也使用 **golangci-lint** 来规范go代码的编写。
+**Reviewbot** 也使用 **golangci-lint** 来规范 go 代码的编写。
 
-默认情况下, **Reviewbot** 使用以下命令来检查go代码:
+默认情况下, **Reviewbot** 使用以下命令来检查 go 代码:
 
 ```bash
-golangci-lint run -D staticcheck
+golangci-lint run -D staticcheck --timeout=5m0s --allow-parallel-runners=true
 ```
 
 :::info
-考虑到充分利用golangci-lint的特性，我们计划开启更多的linters。
+考虑到充分利用 golangci-lint 的特性，我们计划开启更多的 linters。
 详情参见: https://github.com/qiniu/reviewbot/issues/116
 :::

--- a/internal/linters/go/golangci_lint/golangci_lint.go
+++ b/internal/linters/go/golangci_lint/golangci_lint.go
@@ -18,7 +18,8 @@ func init() {
 
 func golangciLintHandler(log *xlog.Logger, a linters.Agent) error {
 	if linters.IsEmpty(a.LinterConfig.Args...) {
-		a.LinterConfig.Args = append([]string{}, "run")
+		// refer: https://github.com/qiniu/reviewbot/issues/146
+		a.LinterConfig.Args = append([]string{}, "run", "--timeout=5m0s", "--allow-parallel-runners=true")
 	}
 
 	return linters.GeneralHandler(log, a, golangciLintParse)

